tools/SetupFlow/DevHome.SetupFlow/Services/WinGet/Operations/WinGetGetPackageOperation.cs

```diff
@@ -45,7 +45,8 @@ public async Task<IList<IWinGetPackage>> GetPackagesAsync(IList<Uri> packageUris
 
         // Get packages grouped by catalog
         var getPackagesTasks = new List<Task<List<IWinGetPackage>>>();
-        foreach (var parsedUrisGroup in GroupParsedUrisByCatalog(packageUrisToQuery))
+        var groupedParsedUris = packageUrisToQuery.GroupBy(p => p.CatalogName).Select(p => p.ToList()).ToList();
+        foreach (var parsedUrisGroup in groupedParsedUris)

```

Comment:
NIT: add `.where` to `groupedParsedUris` in the `foreach` to reduce the indentation level inside the loop by one.